test/unittests/test_webrockets_backend.py

```
@@ -0,0 +1,404 @@
+        return func
+
+
+class _FakeRoute:
+    """Mimics webrockets WebsocketRoute."""
+
+    def __init__(self):
+        self._connect_decorator = _FakeConnectDecorator()
+        self._receive_handler = None
+        self._disconnect_handler = None
+
+    def connect(self, when="after"):
```

Ruff (0.15.5)

[warning] Unused method argument: `when`

(ARG002)